hapi/src/api/account.api.js

```diff
@@ -435,19 +435,26 @@ const verifyEmail = async ({ code }) => {
   }
 }
 
-const login = async ({ account, secret }) => {
+const login = async ({ account, password }) => {
```

Comment:
@adriexnet, could you check if this security flow is right, any doubt about the code, please let me know

Reply:
Just to add something else, I'm gonna add another card to solve the way the secret hash is been created, to make it rest to the backend side instead of the frontend side